addons/hr_holidays/migrations/13.0.1.5/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,80 @@
+---Models in module 'hr_holidays'---
+obsolete model hr.holidays.summary.dept [transient]
+# NOTHING TO DO
+
+---Fields in module 'hr_holidays'---
+hr_holidays  / hr.employee              / leave_manager_id (many2one)   : NEW relation: res.users
+# NOTHING TO DO
+hr_holidays  / hr.leave                 / request_hour_from (selection) : selection_keys is now '['0', '0.5', '1', '1.5', '10', '10.5', '11', '11.5', '12', '12.5', '13', '13.5', '14', '14.5', '15', '15.5', '16', '16.5', '17', '17.5', '18', '18.5', '19', '19.5', '2', '2.5', '20', '20.5', '21', '21.5', '22', '22.5', '23', '23.5', '3', '3.5', '4', '4.5', '5', '5.5', '6', '6.5', '7', '7.5', '8', '8.5', '9', '9.5']' ('[-24, -23, -22, -21, -20, -19, -18, -17, -16, -15, -14, -13, -12, -11, -10, -9, -8, -7, -6, -5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23]')
+hr_holidays  / hr.leave                 / request_hour_to (selection)   : selection_keys is now '['0', '0.5', '1', '1.5', '10', '10.5', '11', '11.5', '12', '12.5', '13', '13.5', '14', '14.5', '15', '15.5', '16', '16.5', '17', '17.5', '18', '18.5', '19', '19.5', '2', '2.5', '20', '20.5', '21', '21.5', '22', '22.5', '23', '23.5', '3', '3.5', '4', '4.5', '5', '5.5', '6', '6.5', '7', '7.5', '8', '8.5', '9', '9.5']' ('[-24, -23, -22, -21, -20, -19, -18, -17, -16, -15, -14, -13, -12, -11, -10, -9, -8, -7, -6, -5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23]')
+# DONE: post-migration: mapped values
+hr_holidays  / hr.leave.allocation      / accrual (boolean)             : DEL
+hr_holidays  / hr.leave.allocation      / manager_id (many2one)         : NEW relation: hr.employee
+# NOTHING TO DO
+hr_holidays  / hr.leave.allocation      / allocation_type (selection)   : NEW required, selection_keys: ['accrual', 'regular'], req_default: function, hasdefault
+# DONE: post-migration: filled the field when empty
+hr_holidays  / hr.leave.type            / allocation_notif_subtype_id (many2one): NEW relation: mail.message.subtype, hasdefault
+hr_holidays  / hr.leave.type            / categ_id (many2one)           : DEL relation: calendar.event.type
+hr_holidays  / hr.leave.type            / code (char)                   : NEW
+hr_holidays  / hr.leave.type            / create_calendar_meeting (boolean): NEW hasdefault
```

Comment:
post-migration: create_calendar_meeting = False if categ_id was not filled (it's True by default)